dataprofiler/labelers/data_processing.py

```diff
@@ -2175,12 +2175,12 @@ def process(self, data, labels=None, label_mapping=None, batch_size=None):
         """Preprocess data."""
         results = {}
         for iter_value, value in enumerate(data):
-            if data[iter_value][0] > self._parameters["positive_threshold_config"]:
+            if labels[iter_value][0] > self._parameters["positive_threshold_config"]:
```

Comment:
typo fix after realizing `labels` is the parameter that represents the output from the model component